Tests/Functional/Bundle/AclBundle/Entity/Car.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\AclBundle\Tests\Functional\Bundle\TestBundle\Entity;
```

Comment:
Hmm I'm having autoloading issues on Travis but not locally (e.g. all tests run green). I'll take a look at it when at work. It seems like one of the renames didn't work properly.